app/javascript/packs/ajax.js

```diff
@@ -0,0 +1,113 @@
+import { resetTargetMarkers, showTargetMarker } from "./leafletMap";
+
+global.ajaxCall = (
```

Comment:
I would be amazed if you could insert a bit of code comments for more context on the most important parts! ❤️